lib/messenger_platform.rb

```diff
@@ -19,8 +19,8 @@ def image(receiver_id, image_url)
     Api.call(:image, receiver_id, image_url)
   end
 
-  def welcome(page_id, text)
-    Api.call(:welcome, page_id, text)
+  def welcome(template_type, *args)
```

Comment:
I think we should use explicit name instead of *args like template_content to make easier to read?